src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/FindOneAndReplaceCommandResolverTest.java

```diff
@@ -234,7 +234,7 @@ public void idFilterConditionWithOptions() throws Exception {
               "findOneAndReplace": {
                 "filter" : {"_id" : "id"},
                 "replacement" : {"col1" : "val1", "col2" : "val2"},
-                "options" : {"returnDocument" : "after", "upsert": true }
```

Comment:
@ivansenic @maheshrajamani Interestingly enough this minor bug (I think copy-pasted) was hidden by "ignore unknown properties"), `Options` for "findOneAndReplace" do not have it (`findOneAndUpdate` does)